### PR TITLE
Improve gauge chart screen reader accessibility

### PR DESCRIPTION
**Pull request by krkshitij**
## Previous Behavior

The screen reader fails to announce the current value when the meter indicator is focused.

## New Behavior

The screen reader announces the current value when the meter indicator is focused.

https://github.com/microsoft/fluentui/assets/110246001/22489c14-60c6-4213-87c6-c196991a8c4c

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/6ddc1cc914239b1f50cbfab08c549a20b41a201e'>6ddc1cc914239b1f50cbfab08c549a20b41a201e</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx [hunk 1]
```diff
@@ -168,8 +170,7 @@ export class GaugeChartBase extends React.Component<IGaugeChartProps, IGaugeChar
                     y={0}
                     textAnchor="start"
                     className={this._classNames.limits}
-                    role="img"
-                    aria-label={`Max value: ${maxValue}`}
+                    aria-hidden={true}
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx:173]: why remove aria-label

File: packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx [hunk 2]
```diff
@@ -369,11 +372,7 @@ export class GaugeChartBase extends React.Component<IGaugeChartProps, IGaugeChar
           strokeWidth={strokeWidth}
           className={this._classNames.needle}
           transform={`translate(${-this._innerRadius + EXTRA_NEEDLE_LENGTH / 2})`}
-          data-is-focusable={true}
-          onFocus={e => this._handleFocus(e, 'Needle')}
-          onBlur={this._handleBlur}
-          onMouseEnter={e => this._handleMouseOver(e, 'Needle')}
-          onMouseMove={e => this._handleMouseOver(e, 'Needle')}
+          aria-hidden={true}
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx:375]: did not get why this was removed

File: change/@fluentui-react-charting-0d3bc989-e3a8-4d0d-8403-4f09d3ee3c86.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-0d3bc989-e3a8-4d0d-8403-4f09d3ee3c86.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=26682&commitId=ef2cd45ed138d2a047c771a069d172e75ebfbef8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by AtishayMsft** [comment]
Kshitij can you share a video of the improved behavior.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**